cmd/gateway/commands_test.go

```diff
@@ -1,289 +0,0 @@
-package main
```

Comment:
Was this supposed to be deleted?

Reply:
yep, because the tests (without updating them) failed :)